src/current/_includes/v22.1/misc/enterprise-features.md

```diff
@@ -1,10 +1,19 @@
+### Cluster optimization
```

Comment:
Because each of these headings are H3 and "See also" is H2, it is rendering a little strangely on the side TOC.